Tools/EventsDeferring/EventMessage.php

```diff
@@ -16,7 +16,7 @@ class EventMessage extends Message
     const HEADER_EVENT_NAME = 'event_name';
 
     /**
-     * @Assert\Valid(traverse=true, deep=true)
+     * @Assert\Valid(traverse=true)
```

Comment:
We should validate that this doesn't break in Symfony 2.8 but based on what I read I don't think it should affect because this property doesn't have arrays of objects. Here is some documentation I found comparing both https://stackoverflow.com/questions/29775773/symfony2-assert-valid-traverse-vs-deep

Reply:
It is safe. It was already `true` in 2.8: https://github.com/symfony/validator/blob/2.8/Constraints/Valid.php#L30

Good point on removing `traverse`.